### PR TITLE
article for launch screens Android

### PR DESCRIPTION
**Pull request by NickIliev**

### CODE REVIEW

File: core-concepts/creating-launch-screens-android.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+title: Creating Launch Screen and App Icons for Android
+description: How to set up and modify Launch Screen and App Icons for Android
+position: 17
+slug: launch-screen-android
+---
+
+
+# Creating Launch Screens for Android 
+
+Launch screens are essential as they provide user's first experience with your mobile application.
```
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: they provide _a_ users’s first...

File: core-concepts/creating-launch-screens-android.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+Based on [Google's Material Design launch screens guidelines](https://material.google.com/patterns/launch-screens.html#launch-screens-branded-launch) there are two 
+main types of launch screens
```
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: Include a colon at the end of this.

File: core-concepts/creating-launch-screens-android.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+### How to set your launch screen
+
+The default template in NativeScript (created with `tns create myApp`) provides you with predefined 
+**splash_screen.xml** file with the NativeScript logo on a blue background and with sample app icons. 
+To modify that template and create your own launch screen using your own assets and design, you will need to access
+the files located under **app/App_Resources/Android** folder.
+Here is the list of all the files and folders you may need to modify:
+
+* **drawable** folders: In your **app/App_Resources/Android** you will find a number of folders named drawable-X (where x is the different DPI for the different devices)
+These folders will store your properly scaled images for your app icons, launch screens and main application (optional).
```
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: main application?

File: core-concepts/creating-launch-screens-android.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+Here is the full list of the files that ships with the basic NativeScript template.  
+
+    * **colors.xml** - XML file in which the app colors are declared.
+    * **strings.xml** - XML file in which the app string are declared.
+    * **styles.xml** - XML file in which the app styles are declared.
+
+* **values-v21** folder: XML files that contain simple values, such as strings, integers, and colors.
+Used when you need to provide themes supported only on API Level 21+ (e.g. Theme.Material)   
+
+* **AndroidManifest.xml** file: Every application must have an AndroidManifest.XML file (with precisely that name) in its root directory. The manifest file presents essential information about your app to the Android system, information the system must have before it can run any of the app's code.
```
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: its root directory? You might want to mention the NativeScript CLI takes care of transferring this file from `App_Resources/Android` to that location.

File: core-concepts/creating-launch-screens-android.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+    * **colors.xml** - XML file in which the app colors are declared.
+    * **strings.xml** - XML file in which the app string are declared.
+    * **styles.xml** - XML file in which the app styles are declared.
+
+* **values-v21** folder: XML files that contain simple values, such as strings, integers, and colors.
+Used when you need to provide themes supported only on API Level 21+ (e.g. Theme.Material)   
+
+* **AndroidManifest.xml** file: Every application must have an AndroidManifest.XML file (with precisely that name) in its root directory. The manifest file presents essential information about your app to the Android system, information the system must have before it can run any of the app's code.   
+In order to change the default launch screen or create your own follow this steps:
```
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: Might want to add a blank line between this and the previous line. GitHub renders this line as part of the bulleted list and that confused me when reading through this.

File: core-concepts/creating-launch-screens-android.md (hunk 4 above)
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: I’m a little confused how `values`, `values-21`, and `AndroidManifest.xml` are relevant here. You opened this section saying that “Here is the list of all the files and folders you may need to modify”, but you don’t explain how any of these previous three bullets related to that at all. I would recommend either deleting all three, or explaining how specifically these folders and file handle launch screens.

File: core-concepts/creating-launch-screens-android.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+1. Provide your properly scaled images accordingly to the **drawable** folders.
+This folder can be used not only for your launch screen images but also for your app icons and for yours
```
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: This folder can be used not only for your launch screen images, but also for your app icons, as well as your

File: core-concepts/creating-launch-screens-android.md (hunk 4 above)
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: Ah ok, I see where they come into play. You may want to include a sentence that says something like, “let’s look at what the various files and folders in `App_Resources/Android` are, and then look at the specific steps you’ll need to take to change them and configure your launch screen.”

File: core-concepts/creating-launch-screens-android.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+in-app images (you can refer to this resource from your application logic with `"res://image-name"` )
+
+![Setting images in drawable resource folders](../img/launch-screen/android/launch-android-005.png "Setting images in drawable resource folders")
+
+2. Define the colors you want to use in **values/colors.xml** and in **values-v21/colors.xml**
```
Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: Might want to explain what those colors actually do. I’m actually not sure what “title-activity-karma” is myself.

Comment (tjvantoll) [inline core-concepts/creating-launch-screens-android.md]: Same comment applies for the strings.

**Comment by tjvantoll** [comment]
I had a few small comments, but overall this looks good @NickIliev. I love the screenshots 😄

File: core-concepts/creating-launch-screens-android.md (hunk 3 above)
Comment (NickIliev) [inline core-concepts/creating-launch-screens-android.md]: what I meant is "in app images" - will correct this as well

**Comment by vakrilov** [comment]
Screenshots are nice, but is there a way to copy the code form somewhere.

Otherwise: 👍 👍 👍

File: publishing/creating-launch-screens-android.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+- follow Goggle's [Material design](https://material.google.com) rules
+
+## Setting launch screen and App Icons
+
+In NativeScript, your application template (created with `tns create myApp`) ships with a basic launch screen template.
+In this article, we are going to introduce the workflow to create/modify your own launch screen.
+
+### Application resources used for launch screen creation
+
+The default template in NativeScript (created with `tns create myApp`) provides you with predefined
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:34]: with “a” predefined

File: publishing/creating-launch-screens-android.md [hunk 9]
```diff
@@ -0,0 +1,139 @@
...
+## Setting launch screen and App Icons
+
+In NativeScript, your application template (created with `tns create myApp`) ships with a basic launch screen template.
+In this article, we are going to introduce the workflow to create/modify your own launch screen.
+
+### Application resources used for launch screen creation
+
+The default template in NativeScript (created with `tns create myApp`) provides you with predefined 
+**splash_screen.xml** file with the NativeScript logo on a blue background and with sample app icons. 
+To modify that template and create your own launch screen using your own assets and design, you will need to access
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:36]: location under “the” ... folder.

File: publishing/creating-launch-screens-android.md [hunk 10]
```diff
@@ -0,0 +1,139 @@
...
+    * **drawable-xxdpi** - Resources for extra-extra-high-density (xxhdpi) screens (~480dpi).
+    * **drawable-xxxdpi** - Resources for extra-extra-extra-high-density (xxxhdpi) uses (~640dpi). Use this for the launcher icon only.
+
+* **values** folder: XML files that contain simple values, such as strings, integers, and colors.
+Here is the full list of the files that ships with the basic NativeScript template.  
+
+    * **colors.xml** - XML file in which the app colors are declared.
+    * **strings.xml** - XML file in which the app string are declared.
+    * **styles.xml** - XML file in which the app styles are declared. 
+    This is the file which holds your `LaunchScreenTheme` style
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:61]: with chainging --> by changing

File: publishing/creating-launch-screens-android.md [hunk 11]
```diff
@@ -0,0 +1,139 @@
...
+    This is the file which holds your `LaunchScreenTheme` style 
+    which you can customize with chainging the `splash_screen.xml` mentioned above.
+    Once your application is loaded the `LaunchScreenTheme` is changed with the `AppTheme` style.
+
+* **values-v21** folder: XML files that contain simple values, such as strings, integers, and colors.
+Used when you need to provide themes supported only on API Level 21+ (e.g. Theme.Material)   
+
+* **AndroidManifest.xml** file: Every application must have an `AndroidManifest.xml` file (with precisely that name) 
+in its root directory. The manifest file presents essential information about your app to the Android system, i
+nformation the system must have before it can run any of the app's code.
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:70]: i and nformation shouldn’t be on separate lines 😄

File: publishing/creating-launch-screens-android.md [hunk 12]
```diff
@@ -0,0 +1,139 @@
...
+Chaning your application icon file is done with modifying the `android:icon` key in `applcation` tag.
+The default app icon set up:
+
+    `android:icon="@drawable/icon"`
+
+    The code above will look for file named __icon.png__ in drawables folders and will load the properly scaled image for the current device.
+
+    > Note: In `AndroidManifest` you will find the following key
+    >`<meta-data android:name="SET_THEME_ON_LAUNCH" android:resource="@style/AppTheme" />`
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:79]: There should be a period after this sentence.

File: publishing/creating-launch-screens-android.md [hunk 13]
```diff
@@ -0,0 +1,139 @@
...
+    >
+    > This key is used by NativeScript to change your `LaunchScreenTheme` with `AppTheme` when your application is loading. 
+
+### How to setup custom launch screen
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:83]: How to setup “a” custom launch screen

File: publishing/creating-launch-screens-android.md [hunk 14]
```diff
@@ -0,0 +1,139 @@
...
+
+In order to change the default NativeScript launch screen (defined in `drawable-nodpi/splash_screen.xml`) and create your own, follow this steps:
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:85]: these steps

File: publishing/creating-launch-screens-android.md [hunk 15]
```diff
@@ -0,0 +1,139 @@
...
+
+1. Provide your properly scaled images accordingly to the **drawable** folders.
+This folder can be used not only for your launch screen images but also for your app icons and for your
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:88]: comma after “launch screen images”

File: publishing/creating-launch-screens-android.md [hunk 16]
```diff
@@ -0,0 +1,139 @@
...
+in-app images (you can refer to this resource from your application logic with `"res://image-name"` )
+The default template app ships with three images: **icon.png** (used for app icon), **logo.png** (centered sample image)
+and **background.png** (image used to fill the background).
+
+The default `splash_screen.xml` with centered 'logo.png' and filled `background.png`
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:93]: Period

File: publishing/creating-launch-screens-android.md [hunk 17]
```diff
@@ -0,0 +1,139 @@
...
+    <item>
+        <bitmap android:gravity="center" android:src="@drawable/logo" />
+    </item>
+</layer-list>
+```
+
+![Setting images in drawable resource folders](../img/launch-screen/android/launch-android-005.png "Setting images in drawable resource folders")
+
+2. Define the colors you want to use in **values/colors.xml** and in **values-v21/colors.xml**.
+You can set your own colors here and reuse them in splash_screen.xml file for your launch screen (or reuse them in your app).
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:109]: in the splash_screen.xml file

Also you should bold spash_screen.xml for consistency.

File: publishing/creating-launch-screens-android.md [hunk 18]
```diff
@@ -0,0 +1,139 @@
...
+
+![Setting colors in values folders](../img/launch-screen/android/launch-android-002.png "Setting colors in values folders")
+
+3. Define the strings you want to use in **values/strings.xml** and in **values-v21/strings.xml**
+You can set your own string here and reuse them in splash_screen.xml file for your launch screen (or reuse them in your app).
```
Comment (tjvantoll) [inline publishing/creating-launch-screens-android.md:114]: Bold the file name again.

**Comment by NickIliev** [comment]
Thank you @tjvantoll  - PR with all the revision for up-vote here [https://github.com/NativeScript/docs/pull/469](https://github.com/NativeScript/docs/pull/469)

**Pull request merged**